### PR TITLE
Anyone can exempt themselves from deposit limits 

### PR DESCRIPTION
**Pull request by davidsiska-vega**
This is to remove a potential point of centralisation.

### CODE REVIEW

File: non-protocol-specs/0003-NP-LIMI-limits_aka_training_wheels.md [hunk 1]
```diff
@@ -129,14 +130,11 @@ This allows for the listing of specific Ethereum addresses to be able to deposit
          - This is true even if both TXs target different [Vega public keys](./../protocol/0017-PART-party.md)
      - Withdrawing all funds after the first transaction, then placing a valid second deposit transaction that causes total lifetime deposits to exceed `max lifetime deposit` is still rejected
      - A single deposit transaction that is more than `max lifetime deposit` rejected 
-     - `lifetime deposit` is tracked across [checkpoints](./0005-NP-LIMN-limited_network_life.md)
```
Comment (barnabee) [inline non-protocol-specs/0003-NP-LIMI-limits_aka_training_wheels.md:132]: Why is this removed?

**Review by barnabee — COMMENTED** [review]
One question about a removed bullet point, otherwise OK to approve

File: non-protocol-specs/0003-NP-LIMI-limits_aka_training_wheels.md (hunk 1 above)
Comment (davidsiska-vega) [reply non-protocol-specs/0003-NP-LIMI-limits_aka_training_wheels.md:132]: All of this is on the smart contract side; tracking lifetime deposit happens on core side. Seems unrelated. 

Also, the bridge contract has to track lifetime deposit per Vega key and per Ethereum key to enforce the limits so this seems redundant.

Comment (barnabee) [reply non-protocol-specs/0003-NP-LIMI-limits_aka_training_wheels.md:132]: Ok makes sense given the Ethereum contract is unaware of checkpoints.
